@xen-orchestra/lite/src/components/vm/VmsActionsBar.vue

```diff
@@ -54,15 +59,43 @@
       </template>
     </MenuItem>
   </AppMenu>
+  <UiModal
+    v-if="isDeleteModalOpen"
+    :icon="faSatellite"
+    @close="closeDeleteModal"
+  >
+    <template #title>
+      <i18n-t keypath="confirm-delete" tag="div">
```

Comment:
It seems that we need to add `scope="global"` here (probably because we are in a slot).